src/mobu/services/business/base.py

```diff
@@ -199,7 +196,7 @@ async def run_once(self) -> None:
     async def idle(self) -> None:
         """Pause at the end of each business loop."""
         self.logger.info("Idling...")
-        with self.timings.start("idle"):
+        with start_span(op="idle"):
             await self.pause(self.options.idle_time)
```

Comment:
Should we drop all the spawns around idling? The only reason why those were being captured before was that we wanted to see if counting up all the timers roughly equaled the elapsed time to make sure that we weren't avoiding timing any major operations.

Reply:
I think it's nice to see the idle spans in the trace, instead of the time being included into whatever the containing span is.

Reply:
Ah, yeah, that makes sense, it's the same use case as we had before.